src/client/containers/LandingPage/LandingPage.styles.css

```diff
@@ -1,3 +1,5 @@
-div {
-  color: #687808;
+.sorting-div {
```

Comment:
Nitpick: Try to avoid class names to refer to the tag that is being used. 
You'll find that you change the HTML to use a `span` and now the class name makes no sense. 
Instead name it after what it does on the page, is it a container, or a box or what is it?